### DISCUSSION TITLE
Can exiv2 detect arithmetic encoding in jpegs?

### DISCUSSION THREAD

**Discussion by eddiezato** [Exiv2/exiv2#2976] [Q&A] [answered]
Something like:
```
> exiftool -encodingprocess 0.jpg
Encoding Process                : Extended sequential DCT, arithmetic coding
```

**Comment by clanmills** [comment, marked as answer]
exiv2 supports Exif, IPTC, XMP, and ICC metadata + thumbnails in many image formats, including JPEG, TIFF and PNG. I believe "encoding process" refers to how the JPEG has compressed the pixel data and is, therefore, not reported by exiv2.  exiv2 has code to navigate different image formats to locate and parse the supported standards and skips everything else including pixels.
